CMakeLists.txt

```diff
@@ -57,3 +57,8 @@ add_custom_target(codespell COMMAND
   -S export-edl.js,CuteLogger,drmingw,node_modules,moc_*,index.theme,*.ts,three.min.js,three.js,jquery.js,rangy-*.js,Makefile,shotcut.pro,*.desktop
 )
 
+add_custom_target(qmlformat COMMAND
+  bash -c [[for f in $(find . -name *.qml); do ~/Qt/5.15.2/gcc_64/bin/qmlformat -i ${f}; done]]
```

Comment:
I would need to find a better way to locate qmlformat. Maybe there is an environment variable that CMake has access to.